### PR TITLE
vcl: fix mq drain if max events threshold hit

### PR DESCRIPTION
**Pull request by florincoras**
Signed-off-by: Florin Coras <fcoras@cisco.com>

Program message queue drain if max events per dispatch threshold is hit to avoid accumulating/losing socket events. 

Risk Level: n/a contrib fix
Testing: n/a

### CODE REVIEW

File: contrib/vcl/source/vcl_interface.cc [hunk 1]
```diff
@@ -77,6 +77,11 @@ void onMqSocketEvents(uint32_t flags) {
       vcl_handle->cb(evts);
     }
   }
+
+  // There might be more unhandled events, so program drain
+  if (!max_events) {
```
Comment (rojkov) [inline contrib/vcl/source/vcl_interface.cc:82]: nit:
```suggestion
  if (max_events != 0) {
```

Comment (florincoras) [reply contrib/vcl/source/vcl_interface.cc]: Done!

**Review by rojkov — APPROVED** [review]
Thanks! Looks good to me

**Pull request merged**